lib/osf-components/addon/components/institutions-widget/component.ts

```diff
@@ -0,0 +1,24 @@
+import { action } from '@ember-decorators/object';
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+import { localClassNames } from 'ember-osf-web/decorators/css-modules';
+import Analytics from 'ember-osf-web/services/analytics';
+import defaultTo from 'ember-osf-web/utils/default-to';
+import styles from './styles';
+import layout from './template';
+
+@localClassNames('InstitutionsWidget')
+export default class InstitutionWidget extends Component {
+    layout = layout;
+    styles = styles;
+
+    @service analytics!: Analytics;
+
+    readOnly: boolean = defaultTo(this.readOnly, false);
+    showModal: boolean = false;
```

Comment:
Something we have started doing in components is to indicate which properties are required paramters, which are optional, and which are "private". The tags-widget demonstrates this:
https://github.com/CenterForOpenScience/ember-osf-web/blob/7304abdc0e494acb2211e4acf1c74ee0f767cff1/lib/osf-components/addon/components/tags-widget/component.ts#L18-L31